crm/themes/inspire/email-welcome.tpl.php

```diff
@@ -13,7 +13,7 @@
     print 'Welcome to ' . "$config_org_name" . '!';
     print '<br /><br />';
     
-    print 'You are receiving this email because you have recently been entered into the ' . "$config_org_name" . ' membership management system, ' . "$config_site_title" . '.';
+    print 'You are receiving this email because you have recently been entered into the ' . "$config_org_name" . ' user management system, ' . "$config_site_title" . '.';
```

Comment:
Are we really changing it to be a user management system? I thought it was crm for makerspace members? And other groups members...